### PR TITLE
Profil card

### PR DESCRIPTION
**Pull request by val-fcn**

### CODE REVIEW

File: frontend/src/components/styles/ProfileCard.scss [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  position: relative;
+
+  .gradientName {
+    position: absolute;
+    top: 250px;
+    left: 0;
+    height: 50px;
+    width: 100%;
+    margin: 0;
+    background-image: linear-gradient(to bottom, #00b0f000, #00b0f0);
```
Comment (TheoBoschetti) [inline frontend/src/components/styles/ProfileCard.scss]: You can use variable instead of hex color

**Review by TheoBoschetti — CHANGES_REQUESTED** [review]

File: frontend/src/components/styles/ProfileCard.scss (hunk 1 above)
Comment (val-fcn) [reply frontend/src/components/styles/ProfileCard.scss]: done

**Review by TheoBoschetti — DISMISSED** [review]
ok

**Review by BastienSaulnier — APPROVED** [review]
SnE

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**